khelo/lib/ui/flow/home/home_screen.dart

```diff
@@ -144,38 +142,40 @@ class _HomeScreenState extends ConsumerState<HomeScreen> {
 
   Widget _teamScore(
     BuildContext context,
-    MatchTeamModel team,
+    MatchTeamModel matchTeam,
     int wicket,
-    bool isCurrentlyPlaying,
   ) {
     return Row(
       children: [
         ImageAvatar(
-          initial: team.team.name[0].toUpperCase(),
-          imageUrl: team.team.profile_img_url,
+          initial: matchTeam.team.name[0].toUpperCase(),
+          imageUrl: matchTeam.team.profile_img_url,
           size: 32,
         ),
         const SizedBox(width: 8),
         Expanded(
-          child: Text(team.team.name,
+          child: Text(matchTeam.team.name,
               overflow: TextOverflow.ellipsis,
+              textScaler: TextScaler.noScaling,
```

Comment:
Suggestion: you can use `MediaQuery.withNoTextScaling` as parent of widget main Widget(in which you don't want to scale text) so we don't have to give it at all the text widgets.